### PR TITLE
Addition of Pubnodes

### PR DESCRIPTION
**Pull request by hooftly**
that's better...

I guess there was an update after I forked the repo oops.

### CODE REVIEW

**Comment by wrkzdev** [comment]
`wrks.pubnodes.com`’s server IP address could not be found. Did you mean `wrkz.pubnodes.com` ?

File: wrkzcoin-nodes.json [hunk 1]
```diff
@@ -42,6 +42,13 @@
       "ssl": false,
       "cache": false
     },
+    {
+      "name": "wrkz.pubnodes.com",
+      "url": "wrks.pubnodes.com",
```
Comment (wrkzdev) [inline wrkzcoin-nodes.json]: `wrkz.pubnodes.com` resolved but `wrks.pubnodes.com` not.

Comment (hooftly) [reply wrkzcoin-nodes.json]: LOL man...

yes its supposed to be wrkz :man_facepalming:

Comment (hooftly) [reply wrkzcoin-nodes.json]: Fixed...

**Pull request merged**